doc/source/configuration/magnum-capi.rst

```diff
@@ -1,45 +1,127 @@
+To configure the Magnum service with the Cluster API driver enabled, first ensure that your kayobe-config branch is up to date with |current_release_git_branch_name|.
 
-Ensure that your magnum.conf has the following set:
+Next, copy the CAPI management cluster's kubeconfig file into to your stackhpc-kayobe-config environment (e.g. ``<your-skc-environment>/kolla/config/magnum/kubeconfig``). This file must be Ansible vault encrypted.
 
-.. code-block:: yaml
+The following config should also be set in your stackhpc-kayobe-config environment:
+
+.. code-block:: ini
+    :caption: magnum.conf
 
     [nova_client]
     endpoint_type = publicURL
 
+.. code-block:: yaml
+    :caption: kolla/globals.yml
 
-This is used to generate the application credential config injected into the tenant Kubernetes clusters, such that it is usable from within an OpenStack project, so you can't use the "internal API" end point here.
+    magnum_cluster_api_driver_enabled: true
 
-Control Plane
-==============
-Ensure that the nodes (either controllers or dedicated network hosts) that you are running the magnum containers on have connectivity to the network on which your management cluster has a floating IP (so that the magnum containers can reach the IP listed in the kubeconfig).
+To apply the configuration, run ``kayobe overcloud service reconfigure -kt magnum``.
 
-Magnum Templates
-================
 
-`azimuth-images <https://github.com/stackhpc/azimuth-images>`__ builds the required Ubuntu Kubernetes images, and `capi-helm-charts <https://github.com/stackhpc/capi-helm-charts/blob/main/.github/workflows/test.yaml>`__ CI runs conformance tests on each image built.
+Magnum Cluster Templates
+========================
 
-Magnum templates can be deployed using `openstack-config <https://github.com/stackhpc/openstack-config>`__. Typically, you would create a fork `<environment>-config` of this repository, move the resources defined in `examples/capi-templates-images.yml` into `etc/openstack-config/openstack-config.yml`, and then follow the instructions in the readme to deploy these.
+The clusters deployed by the Cluster API driver make use of the Ubuntu Kubernetes images built in the `azimuth-images <https://github.com/stackhpc/azimuth-images>`_ repository and then use `capi-helm-charts <https://github.com/stackhpc/capi-helm-charts>`_ to provide the Helm charts which define the clusters based on these images. Between them, these two repositories have CI jobs which regularly build and test images and Helm charts for the latest Kubernetes versions. It is therefore important to update the cluster templates on each cloud regularly to make use of these new releases.
 
+Magnum templates should be defined within an existing client-specific `openstack-config <https://github.com/stackhpc/openstack-config>`_ repository.
 
+TODO: Add more info here once we decide how to manage template updates in openstack-config.
```

Comment:
Should this be commented out, or are you going to add this before we merge?

Reply:
Was planning to add before merge, once https://github.com/stackhpc/openstack-config/pull/21 is merged